website/reference/all-config.md

```diff
@@ -52,6 +52,20 @@ Ship.js will check `dependencies`, `devDependencies` and `peerDependencies` and
 
 1. Ship.js will only publish the packages from `packagesToPublish`.
 
+## `version`
+
+_used at_: `shipjs prepare`
+
+_default_: `undefined`
+
+If `version` is defined, Ship.js will skip all other validation and creation of changelog. It will only run `version` hook.
```

Comment:
Eventually it would be nice to list out exactly what Ship.js usually does for you and _won't_ do if you use `version`, just so you know exactly what you should handle yourself.

```suggestion
When provided, Ship.js skips all other validation and creation of changelog, and only runs `version` instead.
```

Reply:
```suggestion
When provided, Ship.js skips all other validation and creation of changelog, and only runs `version` instead. The steps you need to recreate in this hook are:
- decide on the next version (for each package)
- validate the conditions in "shouldPrepare"
- update version numbers
- updating or writing changelog
```